js/constants/appConstants.js

```diff
@@ -192,6 +192,7 @@ const appConstants = {
   APP_ON_LEDGER_PIN_PUBLISHER: _,
   APP_ON_LEDGER_NOTIFICATION_INTERVAL: _,
   APP_ON_LEDGER_MEDIA_DATA: _,
+  APP_ON_FUZZING: _,
```

Comment:
Could this be changed to something like `APP_ON_LEDGER_FUZZING`?

Reply:
done